src/static/index.scss

```diff
@@ -1,9 +1,35 @@
 ////declare your variables BEFORE importing bootstrap
-$dark: #02233a;
+$dark-theme: "dark-theme";
+$light-theme: "light-theme";
+
+$dark: #001030;
 $primary: #3766b4;
 $secondary: #eef6ff;
+$secondary-hovered: #b2ceec;
 $custom-color: #0f5f35;
 $nav-link-color: #969696;
+$general-background: #FAFAFA;
+
+$themes: (
+        $light-theme: (
+          $dark: #001030,
+          $primary: #3766b4,
+          $secondary: #eef6ff,
+          $secondary-hovered: #b2ceec,
+          $custom-color: #0f5f35,
+          $nav-link-color: #969696,
+          $general-background: #FAFAFA,
+        ),
+        $dark-theme: (
+          $general-background: #040916,
+          $dark: #ffffff,
```

Comment:
Can you change this to an off white?

We try to avoid completely white / black in web design. Notice how even in light theme `$dark` is not #000, but an off black as well.